spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -129,7 +129,7 @@ static class ReporterConfiguration {
 		@Bean
 		@ConditionalOnMissingBean
 		@ConditionalOnBean(Sender.class)
-		AsyncReporter<Span> spanReporter(Sender sender, BytesEncoder<Span> encoder) {
+		Reporter<Span> spanReporter(Sender sender, BytesEncoder<Span> encoder) {
```

Comment:
I think this could be considered a bug and, therefore, it should be handled separately. Could you please split this out into a new PR, along with a test that verifies that the auto-configured `spanReporter` bean backs off when you define a custom `Reporter` bean?

Reply:
OK !

Reply:
submit: https://github.com/spring-projects/spring-boot/pull/35455